### PR TITLE
Plugin logging improvements

### PR DESCRIPTION
**Pull request by dimkouv**
- Include oracleID in all the logs
- Clean up existing logs
- Fix some invalid logs / bugs

### CODE REVIEW

File: commit/merkleroot/outcome.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ func (w *Processor) Outcome(
 	tStart := time.Now()
 	outcome, nextState := w.getOutcome(prevOutcome, query, aos)
 	w.lggr.Infow("Sending Outcome",
-		"outcome", outcome, "oid", w.oracleID, "nextState", nextState, "outcomeDuration", time.Since(tStart))
+		"outcome", outcome, "nextState", nextState, "outcomeDuration", time.Since(tStart))
```
Comment (makramkd) [inline commit/merkleroot/outcome.go:32]: Why remove oracle ID from the fields?

File: commit/plugin.go [hunk 2]
```diff
@@ -69,6 +69,10 @@ func NewPlugin(
 	reportingCfg ocr3types.ReportingPluginConfig,
 	rmnConfig rmn.Config,
 ) *Plugin {
+	lggr = logger.Named(lggr, "CommitPlugin")
+	lggr = logger.With(lggr, "donID", donID, "oracleID", reportingCfg.OracleID)
```
Comment (makramkd) [inline commit/plugin.go:73]: Nice, looks good.

File: commit/plugin.go [hunk 3]
```diff
@@ -56,7 +56,7 @@ type Plugin struct {
 func NewPlugin(
 	_ context.Context,
 	donID plugintypes.DonID,
-	nodeID commontypes.OracleID,
+	oracleID commontypes.OracleID,
 	oracleIDToP2pID map[commontypes.OracleID]libocrtypes.PeerID,
```
Comment (makramkd) [inline commit/plugin.go:60]: This has been bothering me for a while, can this be renamed to `oracleIDToP2PID`?

File: commit/plugin.go [hunk 4]
```diff
@@ -129,7 +133,7 @@ func NewPlugin(
 
 	return &Plugin{
 		donID:               donID,
-		nodeID:              nodeID,
+		nodeID:              oracleID,
```
Comment (makramkd) [inline commit/plugin.go:136]: Would be nice to rename this field in the struct to `oracleID` to be consistent.

File: commit/plugin.go [hunk 5]
```diff
@@ -129,7 +133,7 @@ func NewPlugin(
...
 		oracleIDToP2pID:     oracleIDToP2pID,
```
Comment (makramkd) [inline commit/plugin.go:137]: And this one to `oracleIDToP2PID`

**Review by makramkd — APPROVED** [review]

File: commit/plugin.go (hunk 3 above)
Comment (dimkouv) [reply commit/plugin.go:60]: yeah, we also don't want to provide it twice, it already in the cfg

File: commit/plugin.go (hunk 4 above)
Comment (dimkouv) [reply commit/plugin.go:136]: will do in a different pr

File: commit/merkleroot/outcome.go (hunk 1 above)
Comment (dimkouv) [reply commit/merkleroot/outcome.go:32]: It's part of base logger now

**Pull request merged**